app/__init__.py

```diff
@@ -55,7 +55,7 @@
 app = Flask(__name__)
 
 # We activate all log levels and prevent logs from showing twice.
-app.logger.setLevel(logging.DEBUG)
+app.logger.setLevel(logging.INFO)
```

Comment:
@rokroskar there is already `.Values.global.debug` so if this is true then that level will be `DEBUG`. Otherwise it will default to INFO (either when not set or when debug is false).

What do you think?

Reply:
There is [this PR](https://github.com/SwissDataScienceCenter/renku/pull/2743) but afaik it doesn't currently exist?